toolchain/check/testdata/basics/no_prelude/multifile_raw_ir.carbon

```diff
@@ -92,27 +94,29 @@ fn B() {
 // CHECK:STDOUT:   generics:        {}
 // CHECK:STDOUT:   generic_instances: {}
 // CHECK:STDOUT:   types:
-// CHECK:STDOUT:     type0:           {constant: template instNamespaceType, value_rep: {kind: copy, type: type0}}
-// CHECK:STDOUT:     type1:           {constant: template inst+4, value_rep: {kind: none, type: type2}}
-// CHECK:STDOUT:     type2:           {constant: template inst+5, value_rep: {kind: none, type: type2}}
-// CHECK:STDOUT:     type3:           {constant: template inst+10, value_rep: {kind: none, type: type2}}
+// CHECK:STDOUT:     typeTypeType:    {kind: copy, type: typeTypeType}
+// CHECK:STDOUT:     typeError:       {kind: copy, type: typeError}
+// CHECK:STDOUT:     type7:           {kind: copy, type: type7}
+// CHECK:STDOUT:     type13:          {kind: none, type: type14}
+// CHECK:STDOUT:     type14:          {kind: none, type: type14}
+// CHECK:STDOUT:     type19:          {kind: none, type: type14}
```

Comment:
Why does this end up with 3 references back to type14 (the function, I believe)? I see it's there before, so it's not a regression, just not sure why this happens.

Reply:
From the new output:
```
// CHECK:STDOUT:     'type(inst+4)':    {kind: none, type: type(inst+5)}
// CHECK:STDOUT:     'type(inst+5)':    {kind: none, type: type(inst+5)}
// CHECK:STDOUT:     'type(inst+10)':   {kind: none, type: type(inst+5)}
```
... which we can look up in the instructions list ...
```
// CHECK:STDOUT:     'inst+4':          {kind: FunctionType, arg0: function0, type: typeTypeType}
// CHECK:STDOUT:     'inst+5':          {kind: TupleType, arg0: type_block0, type: typeTypeType}
// CHECK:STDOUT:     'inst+10':         {kind: FunctionType, arg0: function1, type: typeTypeType}
```
... so these are two function types whose value representation is an empty tuple type, plus the empty tuple type itself.